openvino_tensorflow/CMakeLists.txt

```diff
@@ -153,6 +153,7 @@ else()
   install(FILES ${CMAKE_INSTALL_PREFIX}/../ocm/OCM/${OCM_LIB} DESTINATION "${OVTF_INSTALL_LIB_DIR}")
 endif()
 
+# [TODO] Add AUTO, HETERO, and TF FE libs to this list when support is added
```

Comment:
not exactly a todo right, they would be anyway done with when these features get added.

Reply:
Removed